testprogs/TESTrun

```diff
@@ -10067,6 +10095,21 @@ my %apply_blocks = (
 		expr => 'outbound',
 		results => [46],
 	},
+	vntag => {
+		savefile => 'vntag.pcap',
+		expr => 'vntag',
+		results => [65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535],
```

Comment:
In this type of test the way to demonstrate that the new keyword works as expected (i.e. it does not always return true) is to try the `vntag` keyword on both packets that test positive and packets that test negative. One way would be to have non-VNTag packets in the same savefile and another would be to have another test that uses a different savefile and results in all-zeroes.

Reply:
I was hoping vntag_vlan_dest_ip would cover that sufficiently, but that's not entirely true as that combines testing VNTag identification with the 6 byte jump. That would pass if the code called everything VNTag and jumped the bytes regardless. Added non-VNTag packets to the end of vntag.pcap (and renamed it to reflect the mixed nature).